src/app/components/layout/FooterComponent.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from "react";
+
+export class FooterComponent extends Component {
+    render() {
+        return (
+            <footer className="ons-footer grid grid--justify-space-around padding-bottom--2">
+                <div className="ons-footer__body grid__col-8">
+                    <div className="grid__col-12">
+                        <svg className="ons-svg-logo" xmlns="http://www.w3.org/2000/svg" width="197" height="19" viewBox="33 2 552 60">
```

Comment:
Extract svg to separate logo file please, this comment was left before in PR you cancelled